Calculator.Tests/Calculator.Tests/CalculatorTests.cs

```diff
@@ -56,116 +62,97 @@ public void AfterEachTest()
         [Test]
         public void PositiveTestCalculator365days()
         {
-            browser.FindElement(By.XPath("//td[2]//input[@id = 'amount']")).SendKeys("100");
-            browser.FindElement(By.XPath("//input [@id = 'percent']")).SendKeys("10");
-            browser.FindElement(By.XPath("//input [@id = 'term']")).SendKeys("365");
-            browser.FindElement(By.XPath("//input[@type][2]")).Click();
+            new CalculatorPage(browser)
```

Comment:
If you use the same page few times, create a variable for this page and use it.

```suggestion
            CalculatorPage calculatorPage = new CalculatorPage(browser)
            calculatorPage
```

Reply:
sorry, could not understand this, let's discuss

Reply:
i did it as well as I could understand you, let's discuss on call
```CalculatorPage calculatorPage = new CalculatorPage(browser);
            calculatorPage
                .EnterCalculatorData("100", "10", "365", true);

            calculatorPage
                .Calculate();
```